instrumentation/oshi/README.md

```diff
@@ -3,3 +3,22 @@
 | System property                                           | Type    | Default | Description              |
 |-----------------------------------------------------------| ------- | ------- |--------------------------|
 | `otel.instrumentation.oshi.experimental-metrics.enabled`  | Boolean | `false` | Enable the OSHI metrics. |
+
+# Using OSHI with OpenTelemetry Java agent
+
+Step1: Downloading OSHI Jar: 
+
+You can download the OSHI jar file from the official Maven repository 
+https://search.maven.org/search?q=g:com.github.oshi%20AND%20a:oshi-core
+
+
+Step2: Adding OSHI Jar to Classpath:
+
+Once you've downloaded the OSHI jar file, you need to add it to the classpath of your Java application.
+Agent loads oshi classes from system class loader so oshi-core jar should be placed on the class path.
+
+
+Conclusion: By following the steps mentioned above, you can manually download the OSHI jar file and add it to the system classpath for your Java applications, enabling access to system hardware information and monitoring capabilities provided by OSHI.
+
+
+
```

Comment:
```suggestion
Download oshi-core jar from https://search.maven.org/search?q=g:com.github.oshi%20AND%20a:oshi-core and place it on the class path. OpenTelemetry Java agent uses system class loader to load classes from the oshi-core jar that are used for the metrics.
```

Reply:
@trask would this be enough?

Reply:
I think so. There may be some complexity for people around "system class loader", e.g. I think can't just drop it into a spring boot app, but we can deal with that separately if anyone reports an issue about it.

Reply:
hi, 
is there any update on merging this PR?